ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -1606,6 +1606,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-sdn
+- as: azure-ipi-ultrassd-disks-p3-f28
+  cron: 2 3 25 * *
```

Comment:
The cron entry should be generated via https://raw.githubusercontent.com/openshift/release/master/ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh.
```
$ generate-cron-entry.sh azure-ipi-ultrassd-disks-p3-f28 openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml
cron: 22 13 27 * *
```

Reply:
ok , was not aware of this , thanks